### PR TITLE
pattern/dedicated-community-leader

### PR DESCRIPTION
**Pull request by nyeates**
Doing this in part to test out how to get some content into the
base of our repo but still have a pull-based review process

### CODE REVIEW

**Comment by gruetter** [comment]
thanks for removing the comments, @nyeates !

File: dedicated-community-leader.md [hunk 1]
```diff
@@ -38,42 +38,28 @@ Community work - especially communication - make up for a significant percentage
...
 - inspires community members.
-Empower the community leader to dedicate 100 % of his time to community work including communication and development. 
 
-**Review comments**
-- (**Done**) important point is that community leader is excellent networker - not necessarily already networked
+Empower the community leader to dedicate 100 % of his time to community work including communication and development. 
 
 ## Resulting Context
 
 A community leader with the properties described above will lend a face and embody the companies commitment to InnerSource. It will make it more likely that other associates in his network will follow his lead and contribute to InnerSource. Over time, he will be able to build up a stable core team of developers and hence increase the chances of success for the InnerSource project. By convincingly a large enough audience within his company of the potential of InnerSource, he will make an important contribution to changing the company culture towards a community culture.
```
Comment (nyeatestest) [inline dedicated-community-leader.md]: Testing testing; what if I didn't like this paragraph. Can I comment on it? Looks like it!

File: dedicated-community-leader.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ _Dedicated Community Leader_
...
+Consider the following story. Company A wants to initiate an InnerSource initiative in order to foster collaboration across organizational boundaries. They have decided to start with an experimental phase with limited scope. Management has selected a suitable pilot topic for the first InnerSource community and expects contributions from many business units across the organization. Company A has nominated a new hire to head the community for 50 % of his work time, because he was not yet 100 % planned for. After 6 months, the community has received only a few contributions, most of which are from a single business unit. Company A replaces the community leader with someone who has a longer history in the company, this time for only 30 % of his time. After another 6 months, the number of contributions has picked up only  marginally. Company A is no longer convinced that InnerSource helps to achieve their goal of increased, cross divisional collaboration and abandons InnerSource. 
 
 **Review Comments**
 - (**Done**) add summary about underlying core problem present in this story at the beginning (possibly boldfaced)
```
Comment (nyeatestest) [inline dedicated-community-leader.md]: Test test; Yes lets get this done.

**Review by NewMexicoKid — APPROVED** [review]
The proposed changes look okay.

**Pull request merged**